auctions/templates/bid_list.html

```diff
@@ -27,7 +26,9 @@
                     {% if bid.ask_met %}
                         data-tooltip aria-haspopup="true" class="has-tip" data-disable-hover="false" title="Your ask has been met"
                     {% endif %}
-                    >{{ bid.ask }}</span>{% if bid.ask_met %} <i class="fi-check"></i>{% endif %}</td>
+                    >{{ bid.ask }}</span>{% if bid.ask_met %} <i class="fi-check"></i>{% endif %}
+                </td>
+            {% for offer in bid.offers %}
```

Comment:
Should loop inside the `<td>` so we don't draw multiple columns in the outer table.